test/integration/test_integration.py

```diff
@@ -60,7 +60,7 @@ def test_two_three_nevr_passed(nevr):
     assert run_task(nevr)['python-versions.two_three'].outcome == 'PASSED'
 
 
-@pytest.fixture()
+@pytest.fixture(scope="session")
```

Comment:
The problem here is that `run_task` prints. By default, pytest will only print the captured output on a failed test. If first test using this passes, and another one fails, it will not be printed now. The idea how to make it useble is:

 1. Have a ("private", underscored) session scoped fixture that runs the task but also returns the captured stdout/stderr
 2. Have a ("public") normal scoped fixture that uses the one above and prints the output

I have done this before and con do it, if you like.